tests/hostcfgd/sample_output/PASSWORD_HARDENING_enable_digits_class/login.defs

```diff
@@ -157,9 +157,9 @@ UMASK		022
 #	PASS_MIN_DAYS	Minimum number of days allowed between password changes.
 #	PASS_WARN_AGE	Number of days warning given before a password expires.
 #
-PASS_MAX_DAYS	99999
+PASS_MAX_DAYS	0
 PASS_MIN_DAYS	0
-PASS_WARN_AGE	7
+PASS_WARN_AGE	0
```

Comment:
This fix a UT, however the code change in this PR not cover by UT: 

when feature enabled but expiration/expiration_warning not defined in CONFIG_DB,  the value of PASS_MAX_DAYS/PASS_WARN_AGE should be -1

Reply:
the definition was:
when the feature is disabled -> move to Linux default which is 7.

This is in motivation to be align to Debian defaults - to customers that do not want this feature.

Reply:
Add a UT to cover this feature enabled but 'expiration' and 'expiration_warning' not defined in CONFIG_DB case.

Reply:
good idea, I will raise this point. 
About this PR, pls can you approve - so, this PR will not postpone the fix-issue.

Reply:
@liuh-80  kind reminder to merge/review